src/apps/backend/modules/openai/openai-service.ts

```diff
@@ -0,0 +1,10 @@
+import OpenAIAdapter from './rest-api/openai-adapter';
+import { Nullable } from './types';
+
+export default class OpenAIService {
+  public static async generateDocumentation(
```

Comment:
Is it possible to make this method a bit more generic that accepts like a prompt and return a response?